homeassistant/components/wiser/switch.py

```diff
@@ -0,0 +1,293 @@
+"""
```

Comment:
Limit the initial PR to a single platform, please remove this platform.
See: <https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

Reply:
Hi Frenck,

Thanks for the review so far. This integration has been developed over sometime (since jan 19) and published via the HACS store. Due to popular demand they've asked me to publish it as part of HA natively and hence it is a little feature filled compared to others.. Its actually version 20 :-)

Do you want me to do an initial PR with just one of these and then issue multiple PRs for the each of the additional components?

Just want to check if thats what you mean.
thanks
Angelo
For reference the original base git repo is https://github.com/asantaga/wiserHomeAssistantPlatform